pkgs/by-name/ap/apotris/package.nix

```
@@ -0,0 +1,101 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  meson,
+  ninja,
+  python3,
+  cmake,
+  pkg-config,
+  xxd,
+  libopus,
+  libogg,
+  zlib,
+  SDL2,
+  SDL2_mixer,
+  libopenmpt,
+  libXrandr,
+  libXext,
+  libXfixes,
+  libXcursor,
+  libXi,
+  libXScrnSaver,
+  makeDesktopItem,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
let
  version = "4.0.2"
in
stdenv.mkDerivation {
```

Could you remove the `rec` keyword? (ref: https://nix.dev/guides/best-practices#recursive-attribute-set-rec)